python/cudf/cudf/core/series.py

```diff
@@ -512,7 +512,9 @@ def copy(self, deep=True):
     def __copy__(self, deep=True):
         return self.copy(deep)
 
-    def __deepcopy__(self):
+    def __deepcopy__(self, memo={}):
```

Comment:
We likely shouldn't default this to an empty dict as the default arg is only initialized once and can then be mutated. Can we default to `None` instead?